ci/build-tumbleweed.sh

```diff
@@ -1,18 +1,13 @@
 #!/bin/sh
 set -e
+. ./ci/build-common.sh
 
-meson setup build \
-  --werror        \
-  -Dc_args="-Wno-error=deprecated -Wno-error=deprecated-declarations" \
-  -Db_sanitize=address,undefined \
```

Comment:
Why is sanitizer removed?

Reply:
Should have been in common.

Reply:
I didn't want to enable it for all of them, but sure that works too. I think mingw gcc doesn't support sanitizers fwiw.

Reply:
Nevermind, seems to be more trouble than it's worth trying to do it for everything.